commcare_connect/opportunity/tables.py

```diff
@@ -549,3 +553,285 @@ def date_with_time_popup(table, date):
         date.strftime("%d %b, %Y"),
         date.strftime("%d %b %Y, %I:%M%p"),
     )
+
+
+def header_with_tooltip(label, tooltip_text):
+    return mark_safe(
+        f"""
+        <div class="relative inline-flex justify-center items-center group cursor-default">
+            <span>{label}</span>
+            <i class="fa-regular fa-circle-question text-xs text-slate-400 ml-1 cursor-help"></i>
+            <div class="fixed hidden group-hover:block z-50 pointer-events-none -translate-x-[15%] -translate-y-[70%] transform">
+                <div class="absolute top-full left-1/2 -translate-x-1/2 w-0 h-0 border-l-8 border-r-8 border-t-8 border-l-transparent border-r-transparent border-t-white"></div>
+                <div class="relative bg-white w-28 rounded p-2 text-slate-500 text-xs whitespace-normal break-words">
+                    {tooltip_text}
+                </div>
+            </div>
+        </div>
```

Comment:
This could potentially be turned into a component by introducing classes like `tooltip-wrapper, tooltip-content` etc. But this is the only place its used.